pkgs/applications/audio/paulstretch/default.nix

```
@@ -30,8 +30,9 @@ stdenv.mkDerivation {
   ];
 
   patches = [
+    # https://github.com/paulnasca/paulstretch_cpp/12
```

```suggestion
    # https://github.com/paulnasca/paulstretch_cpp/pull/12
```

All are missing the /pull/